### PR TITLE
AddQuestion component

### PR DESCRIPTION
**Pull request by agasior093**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -15,15 +15,12 @@ import { QuestionsContentComponent } from './components/questions-content/questi
...
-  HighlightModule,
-  HIGHLIGHT_OPTIONS,
-  HighlightOptions
-} from 'ngx-highlightjs';
 import { NewQuestionComponent } from './components/new-question/new-question.component';
 import { QuestionViewComponent } from './components/question-view/question-view.component';
 import { TagSubmenuComponent } from './components/tag-submenu/tag-submenu.component';
-
+import { EditorComponent } from './components/editor/editor.component';
+import { AutocompleteComponent } from './components/autocomplete/autocomplete.component';
```
Comment (Xupi) [inline src/app/app.module.ts]: Why no AutoCompleteComponent?

File: src/app/components/autocomplete/autocomplete.component.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  addItem() {
+    this.outputItems.push(this.currentItem.trim());
+    this.currentItem = '';
+    this.outputItemsChanged.emit(this.outputItems);
+  }
+
+  onKeyUp($event) {
+    if (this.currentItem !== undefined && this.currentItem.trim() !== '' && ($event.key === 'Enter' || $event.key === ' ')) {
```
Comment (Xupi) [inline src/app/components/autocomplete/autocomplete.component.ts]: What if someone adds "Ruby on rails"?

File: src/app/components/new-question/new-question.component.ts [hunk 3]
```diff
@@ -1,35 +1,85 @@
...
+      this.selectedTags = [];
+    }, err => {
+      this.messages = { type: 'danger', content: this.parseErrors(err) };
+      this.loading = false;
+    });
+  }
+
+  private parseErrors(error: any): string[] {
+    if (error.error && error.error.errors) {
+      return error.error.errors.map(err => err.defaultMessage);
```
Comment (Xupi) [inline src/app/components/new-question/new-question.component.ts]: xD

File: src/app/components/autocomplete/autocomplete.component.ts (hunk 2 above)
Comment (agasior093) [reply src/app/components/autocomplete/autocomplete.component.ts]: Tags shouldn't containt spaces but - instead

File: src/app/components/new-question/new-question.component.ts (hunk 3 above)
Comment (agasior093) [reply src/app/components/new-question/new-question.component.ts]: xD

**Pull request merged**